tac/agents/participant/v1/agent.py

```diff
@@ -145,6 +145,7 @@ def start(self, rejoin: bool = False) -> None:
             self.oef_handler.rejoin = rejoin
             super().start()
             self.oef_handler.rejoin = False
+            self.stop()
```

Comment:
baseline agent v1 didn't stop the execution with `self.stop`, but setting `self.liveness._is_stopped` to `True`. However, that makes `mailbox.disconnect()` never being called.